ansible_collections/arista/avd/roles/eos_designs/templates/documentation/fabric-topology.j2

```diff
@@ -1,4 +1,4 @@
-Node Type,Node,Node Interface,Peer Type,Peer Node,Peer Interface
+Node Type,Node,Node Interface,Peer Type,Peer Node,Peer Interface,Provisioned in CloudVision
```

Comment:
Instead of documenting the `is_deployed` state per link, when it is really a device parameter for one or the other device, I think we should just show the per-link `shutdown` with True/False as value

Reply:
agreed. Rather than having to work out whether true/false means shutdown = true or enabled = true, I'm going to make this explicit. The cell will contain "Enabled" or "Shutdown".

Reply:
Using to specific strings makes it easier for humans but harder for computers. This is a CSV, so more likely to be consumed by a script of some sort. Personally I have not used CSVs anywhere, but I know @carlbuchmann has, so  hopefully he can chip in.

Reply:
This is now `Node Interface Enabled: True|False`